lib/splitclient-rb/engine/evaluator/splitter.rb

```diff
@@ -50,7 +48,12 @@ def count_hash(key, seed, legacy)
       end
 
       def murmur_hash(key, seed)
-        Digest::MurmurHash3_x86_32.rawdigest(key, [seed].pack('L'))
+        case RUBY_PLATFORM
```

Comment:
what's the runtime penalty?

Can we avoid making this case statement on every evaluation and bind it one time during start up?

Reply:
Yes, thank you, I'll change that

Reply:
@patricioe fixed